Money/Money.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+
+namespace OperatorOverloadingModel
+{
+    public class Money
+    {
+        private Double Amount;
```

Comment:
Naming Convention: private variables starts with underscore and then camel casing.